extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java

```diff
@@ -302,6 +325,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
     /**
      * Override this function if the format of time stamp in the url is not the same format as that form youtube.
      * Honestly I don't even know the time stamp fromat of youtube.
+     *
      * @param regexPattern
      * @return the sime stamp/seek for the video in seconds
      * @throws ParsingException
```

Comment:
below:
// -2 because well the testing system will then know its the regex that failed :/
will consequently know that the regex failed

Reply:
I don't understand what you mean. Can you explain in more detail please?

Reply:
Below this line there is a comment
``// -2 because well the testing system will then know its the regex that failed``
I suggest to reword it to
``// -2 because the testing system will consequently know that the regex failed``

Reply:
Done